plugins/modules/vault_login.py

```diff
@@ -40,10 +41,10 @@
   attributes:
     check_mode:
       support: partial
-      description:
-        - "In check mode, this module will not perform a login, and will instead return a basic structure with an empty token.
+      details:
+        - In check mode, this module will not perform a login, and will instead return a basic structure with an empty token.
           However this may not be useful if the token is required for follow on tasks.
-          It may be better to use this module with C(check_mode=no) in order to have a valid token that can be used."
+        - It may be better to use this module with C(check_mode=no) in order to have a valid token that can be used.
```

Comment:
I'm not sure whether you already updated the documentation to use `true`/`false` in other places (or whether you want to do that), but here is a place where `yes`/`no` is still in use:
```suggestion
        - It may be better to use this module with C(check_mode=false) in order to have a valid token that can be used.
```